content/blog/2023-05-16-comparison-of-ethereum-classic-and-ethereum-monetary-policies/index.md

```diff
@@ -0,0 +1,146 @@
+
+## Algorithmic
+
+ETC’s monetary policy is algorithmic because in 2017 it migrated to a schedule that is set in stone as it establishes the whole future distribution in the software in the form of a supply algorithm.
+
+The supply algorithm is a set of simple and transparent mathematical rules that establish what number of coins will be paid to miners per block, the frequency of the payments, and a schedule that will decrease these payments as time passes by.
+
+ETH’s monetary policy is manual, not algorithmic, because it has been changed several times and will likely change many times more in the future given that proof of stake tends to centralization.
+
+The fact that it has a software algorithm that determines payments to validators does not mean it is algorithmic as long as developers may change the rules whenever they please.
+
+## Capped Supply
+
+ETC has a capped supply of its coin. This means that it’s known what is the total amount of ETC that will ever exist.
+
+Given Ethereum Classic’s frequency of payments to miners, the number of coins it issues per block, and its discount schedule, it will have a maximum supply of ETC of 210,700,000.
+
+ETH does not have a capped supply, but has a two way system with an issuance schedule and a coin burning mechanism. These two combined sometimes produce a net issuance of coins and sometimes a net redemption of coins. It all depends on the amounts of staked ETH by validators and the levels of transaction fees.
+
+However, whether Ethereum has a capped supply or not , or if they burn coins or not, is inconsequential in a centralized network with a manually managed monetary policy because they may change the rules at any time. 
+
+## Block Time
+
+Ethereum Classic’s block time is 13 seconds per block and Ethereum’s is 12 seconds per slot, and a slot is a space where a block fits.
+
+This means that ETC miners create 6,646 blocks per day and ETH validators create 7,200 blocks in the same period.
+
+The block time of a blockchain is important because it establishes how fast the coins will be created, thus regulates the supply (and the burn rate, in the case of ETH). 
+
+By modulating the block times and the number of coins created (or burned) per block then a blockchain can control the monetary policy in time and supply levels. 
+
+## Eras
+
+Ethereum Classic has eras and Ethereum does not.
+
+Eras in ETC are measured in blocks in which the payments to miners, thus the issuance of coins, will be the same for each era.
+
+Ethereum Classic’s eras last 5 million blocks which, at 13 seconds per block, turns out to be more or less 2 years per era.
+
+After each era ETC discounts the payment to miners.
+
+## Discount Per Era
+
+In Ethereum Classic, the discount per era is 20%. 
```

Comment:
Line 93: Fix line spacing; Rewrite for clarity
```suggestion
In Ethereum Classic, the discount per era is 20%. This means that since the genesis block, ETC has paid miners 5 ETC per block in the first era, then 4 ETC, then 3.20, and is now paying 2.56 per block during the current era.
```